dds_cli/user.py

```diff
@@ -156,6 +163,10 @@ def read_token(self):
         LOG.debug("Token retrieved from file.")
         return token
 
+    def file_exists(self):
+        """Returns True if the token file exists."""
+        return self.token_file.is_file()
```

Comment:
Since the  `token_file` is a `Path` I'm thinking we should just use `self.token_file.exists()` instead of the `def file_exists()` function?

Reply:
Yes, it's just that I wanted to expose it to be used in the auth commands. So in the `auth.py` it would be `token_file.token_file.exists()` with your suggestion instead of `token_file.file_exists()` now. I would be fine with both though.